src/Socket/messages-recv.ts

```
@@ -1195,6 +1196,15 @@ export const makeMessagesRecvSocket = (config: SocketConfig) => {
 				})
 				authState.creds.registered = true
 				ev.emit('creds.update', authState.creds)
+				break
+			case 'passkey_prologue_request':
+			case 'crsc_continuation':
+				const passkeyRequest = getPasskeyRequestState(node)
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)